README.md

```diff
@@ -58,7 +60,119 @@ Master and Worker nodes:
+amphora image should be available in Glance. Please refer to documentation in "Installing with
+Cloud Lifecyle Manager" guide in Post Installation->Configuring Loadbalancer as a Service->Section 31.3
+[Setup of prerequisites](https://www.suse.com/documentation/suse-openstack-cloud-8/book_installation/data/sect1_51_chapter_book_installation.html)
+for instructions on how to setup prerequisites which includes steps on "Installing the Amphora Image", "Register the image", and also
+steps on "Setup network, subnet, router, security and IP's" to set up a test "lb_net1" network with "lb_subnet1" subnet.
+e.g.
+```
+openstack network create lb_net1
+openstack subnet create --name lb_subnet1 lb_net1 --subnet-range 172.29.0.0/24 \
+  --gateway 172.29.0.2
+openstack router create lb_router1
+openstack router add subnet lb_router1 lb_subnet1
+openstack router set lb_router1 --external-gateway ext-net
+openstack network list
+
+Please use ID of lb_subnet1 as <SUBNET_ID> in neutron lbaas-loadbalancer-create commands above
+```
+To clean up the environment after the test please delete loadbalancer, router subnet and network
+that were created for the test.
+```
+openstack loadbalancer delete test-lb
+openstack subnet delete lb_subnet1
+openstack router delete lb_router1
+openstack network delete lb_net1
+```
+
+4. Create a stack in Heat by passing the template, environment file, and parameters:
+```
+  $ openstack stack create -t caasp-multi-master-stack.yaml -e caasp-multi-master-environment.yaml \
+    --parameter image=CaaSP-3 caasp-multi-master-stack
+```
+
+5. Floating IP address of the load balancer can be found using the
+   following steps
+
+```
+Openstack CLI:
+openstack loadbalancer list --provider
+OR
+neutron lbaas-loadbalancer-list
+neutron CLI is deprecated and will be removed in the future. Use openstack CLI instead.
++--------------------------------------+------------------------------------+----------------------------------+-------------+---------------------+----------+
+| id                                   | name                               | tenant_id                        | vip_address | provisioning_status | provider |
++--------------------------------------+------------------------------------+----------------------------------+-------------+---------------------+----------+
+| 0d973d80-1c79-40a4-881b-42d111ee9625 | caasp-stack-master_lb-bhr66gtrx3ue | fd7ffc07400642b1b05dbef647deb4c1 | 172.28.0.6  | ACTIVE              | haproxy  |
++--------------------------------------+------------------------------------+----------------------------------+-------------+---------------------+----------+
+
+Openstack CLI:
+openstack loadbalancer show 0d973d80-1c79-40a4-881b-42d111ee9625
+OR
+neutron lbaas-loadbalancer-show 0d973d80-1c79-40a4-881b-42d111ee9625
+neutron CLI is deprecated and will be removed in the future. Use openstack CLI instead.
++---------------------+------------------------------------------------+
```

Comment:
These command outputs are not very readable when rendered.. Can you maybe use multi-line for these?

Reply:
done. can you please check again. Adding closing backticks made it a complete code block

Reply:
You can check the formatting here. Still many places where the code is not in code blocks and the output is not aligned.
https://github.com/SUSE/caasp-openstack-heat-templates/blob/096848975987fe431f08e6eb3ac6e0330d7e61c1/README.md

Reply:
> You can check the formatting here. Still many places where the code is not in code blocks and the output is not aligned.
> https://github.com/SUSE/caasp-openstack-heat-templates/blob/096848975987fe431f08e6eb3ac6e0330d7e61c1/README.md

This seems better https://github.com/SUSE/caasp-openstack-heat-templates/blob/8c9ffe084a42dead95867d8f118c17ab616dc2d9/README.md